test/bin/mirror_registry.sh

```diff
@@ -29,17 +29,8 @@ retry_pull_image() {
 }
 
 get_container_images() {
-    containers=""
-    local -r release_info_rpm=$(find "${IMAGEDIR}/rpm-repos" -name "microshift-release-info-*.rpm" | sort)
-    if [ -z "${release_info_rpm}" ] ; then
-        echo "Error: missing microshift-release-info RPMs"
-        exit 1
-    fi
-    for package in ${release_info_rpm}; do
-        echo "Getting image references from RPM ${package}..."
-        containers="$(rpm2cpio "${package}" | cpio  -i --to-stdout "*release-$(uname -m).json" 2> /dev/null | jq -r '[ .images[] ] | join("\n")')\n${containers}"
-    done
-    echo -n -e "${containers}" | sort -u > "${REGISTRY_CONTAINER_LIST}"
+    local -r container_file=$1
+    sort -u "${container_file}" | uniq > "${REGISTRY_CONTAINER_LIST}"
```

Comment:
I think `sort -u` already produces a unique list, so no need for `uniq` call.

Reply:
Done.